src/include/mutex.i

```diff
@@ -286,6 +286,13 @@ __wt_spin_unlock(WT_SESSION_IMPL *session, WT_SPINLOCK *t)
 	    S2C(session)->stats, lock_##name##_wait_internal);		\
 } while (0)
 
+#define	WT_SPIN_INIT_SESSION_TRACKED(session, t, name) {                \
+	WT_SPIN_INIT_TRACKED(session, t, name);                         \
+	(t)->stat_session_usecs_off = \
```

Comment:
the '\' at the end of this line isn't aligned properly.

Reply:
I tried aligning the \ , but s_all always brings it back to where it is.